setup.py

```diff
@@ -12,6 +12,7 @@
         ('share/ament_index/resource_index/packages',
             ['resource/' + package_name]),
     ],
+    package_data={'': ['py.typed']},
```

Comment:
precisely like this?

```suggestion
    package_data={'rpyutils': ['py.typed']},
```

Reply:
Either is fine since we are not building multiple packages with the same `setup.py`. If you would prefer the version if `rpyutils` in it will update.